src/app/components/admin/admin-collections/admin-collections.component.ts

```diff
@@ -144,6 +145,7 @@ export class AdminCollectionsComponent implements OnInit, OnDestroy {
       collection.name.toLowerCase().includes(this.filterString) ||
       collection.description.toLowerCase().includes(this.filterString)
     );
+    this.filteredCollectionCount = this.filteredCollectionList.length;

```

Comment:
It doesn't look like filteredCollectionCount is used anywhere.  The html file uses filteredCollectionList.length